README

```diff
@@ -1,3 +1,21 @@
-Smiley icons
-============
-GNOME icon theme is distributed under the terms of either GNU LGPL v.3 or Creative Commons BY-SA 3.0 license.
+What is this?
+=============
+
+Chat is an activity used to exchange messages with friends or classmates.
+You can chat about a topic you are studying or you can share something private that happened in your life.
+
+How to use?
+===========
+
+Chat is part of the Sugar desktop.  Please refer to;
+
+* [How to Get Sugar on sugarlabs.org](https://sugarlabs.org/),
+* [How to use Sugar](https://help.sugarlabs.org/),
+* [How to use Chat](https://help.sugarlabs.org/chat.html)
+
+How to upgrade?
```

Comment:
While this is consistent with other similar PRs, is there an advantage of adding `How to Upgrade` in the readme of an activity ?

Reply:
interesting question; :)

yes, for situations when developers/maintainers wants to make sure they have the latest version of activity installed, before trying to reproduce a bug

correct me, if I'm wrong

Reply:
The latest copy of the activity is usually on GitHub before (rather) than ASLO.

Reply:
true.
should be remove it then?

Reply:
Yes, thanks